src/Generate/Export.php

```diff
@@ -0,0 +1,81 @@
+<?php namespace Brackets\AdminGenerator\Generate;
+
+use Illuminate\Support\Str;
+use Symfony\Component\Console\Input\InputOption;
+
+class Export extends ClassGenerator {
+
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $name = 'admin:generate:export';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Generate an export class';
+
+    /**
+     * Path for view
+     *
+     * @var string
+     */
+    protected $view = 'export';
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $force = $this->option('force');
+
+        if(!empty($template = $this->option('template'))) {
+            $this->view = 'templates.'.$template.'.export';
+        }
+
+        if ($this->generateClass($force)){
+            $this->info('Generating '.$this->classFullName.' finished');
+        }
+
+    }
+
+    protected function buildClass() {
+        return view('brackets/admin-generator::'.$this->view, [
+            'exportNamespace' => $this->classNamespace,
+            'modelFullName' => $this->modelFullName,
+            'classBaseName' => $this->exportBaseName,
+            'modelBaseName' => $this->modelBaseName,
+            'modelVariableName' => $this->modelVariableName,
+            'columnsToExport' => $this->readColumnsFromTable($this->tableName)->filter(function($column) {
+                return !($column['type'] == 'text' || $column['type'] == 'boolean' || $column['name'] == "password" || $column['name'] == "remember_token" || $column['name'] == "slug" || $column['name'] == "updated_at" || $column['name'] == "deleted_at");
```

Comment:
@dejwCake sme vraveli, ze `text` polia v exporte praveze chceme nie? aj `boolean` by som nechal - treba to ale asi rozumne preformatovat na true/false alebo take nieco. Aj slug by som nechal. Ostatne je asi OK.